spec/core/AsyncExpectationSpec.js

```diff
@@ -178,40 +178,52 @@ describe('AsyncExpectation', function() {
       });
     });
 
-    it('builds its message correctly when negated', function() {
+  });
+
+  describe('#toBeRejectedWith', function() {
+    it('passes if the promise is resolved to the expected value', function() {
       jasmine.getEnv().requirePromises();
 
+      var actual = Promise.resolve({foo: 42});
```

Comment:
I'm pretty sure this should be `Promise.reject` in order to make the matcher pass under the appropriate circumstances. Should probably also change the test description to match